src/hnsw/build.c

```diff
@@ -91,6 +93,15 @@ static void AddTupleToUsearchIndex(ItemPointer tid, Datum *values, HnswBuildStat
     if(buildstate->usearch_index != NULL) {
         size_t capacity = usearch_capacity(buildstate->usearch_index, &error);
         if(capacity == usearch_size(buildstate->usearch_index, &error)) {
+            double             M = ldb_HnswGetM(index);
+            double             mL = 1 / log(M);
+            usearch_metadata_t meta = usearch_metadata(buildstate->usearch_index, &error);
+            uint32             node_size = UsearchNodeBytes(&meta, meta.dimensions * sizeof(float), (int)(mL + .5));
```

Comment:
where is the `mL+.5` term coming from?
I assume this is the expected node size, given the distribution in the paper. Could you write a note in the comment around it?